data-manipulation/encoding/encode-data-using-add-xor-sub-operations.yml

```diff
@@ -0,0 +1,791 @@
+          - operand[1].number: 0xb9
+          - operand[1].number: 0xba
+          - operand[1].number: 0xbb
+          - operand[1].number: 0xbc
+          - operand[1].number: 0xbd
+          - operand[1].number: 0xbe
+          - operand[1].number: 0xbf
+          - operand[1].number: 0xc0
+          - operand[1].number: 0xc1
+          - operand[1].number: 0xc2
+          - operand[1].number: 0xc3
+          - operand[1].number: 0xc4
+          - operand[1].number: 0xc5
+          - operand[1].number: 0xc6
+          - operand[1].number: 0xc7
+          - operand[1].number: 0xc8
+          - operand[1].number: 0xc9
+          - operand[1].number: 0xca
+          - operand[1].number: 0xcb
+          - operand[1].number: 0xcc
+          - operand[1].number: 0xcd
+          - operand[1].number: 0xce
+          - operand[1].number: 0xcf
+          - operand[1].number: 0xd0
+          - operand[1].number: 0xd1
+          - operand[1].number: 0xd2
+          - operand[1].number: 0xd3
+          - operand[1].number: 0xd4
+          - operand[1].number: 0xd5
+          - operand[1].number: 0xd6
+          - operand[1].number: 0xd7
+          - operand[1].number: 0xd8
+          - operand[1].number: 0xd9
+          - operand[1].number: 0xda
+          - operand[1].number: 0xdb
+          - operand[1].number: 0xdc
+          - operand[1].number: 0xdd
+          - operand[1].number: 0xde
+          - operand[1].number: 0xdf
+          - operand[1].number: 0xe0
+          - operand[1].number: 0xe1
+          - operand[1].number: 0xe2
+          - operand[1].number: 0xe3
+          - operand[1].number: 0xe4
+          - operand[1].number: 0xe5
+          - operand[1].number: 0xe6
+          - operand[1].number: 0xe7
+          - operand[1].number: 0xe8
+          - operand[1].number: 0xe9
+          - operand[1].number: 0xea
+          - operand[1].number: 0xeb
+          - operand[1].number: 0xec
+          - operand[1].number: 0xed
+          - operand[1].number: 0xee
+          - operand[1].number: 0xef
+          - operand[1].number: 0xf0
+          - operand[1].number: 0xf1
+          - operand[1].number: 0xf2
+          - operand[1].number: 0xf3
+          - operand[1].number: 0xf4
+          - operand[1].number: 0xf5
+          - operand[1].number: 0xf6
+          - operand[1].number: 0xf7
+          - operand[1].number: 0xf8
+          - operand[1].number: 0xf9
+          - operand[1].number: 0xfa
+          - operand[1].number: 0xfb
+          - operand[1].number: 0xfc
+          - operand[1].number: 0xfd
+          - operand[1].number: 0xfe
+          - operand[1].number: 0xff
+      - instruction:
+        - mnemonic: add
+        - or:
+          - operand[1].number: 0x01
+          - operand[1].number: 0x02
+          - operand[1].number: 0x03
+          - operand[1].number: 0x04
+          - operand[1].number: 0x05
+          - operand[1].number: 0x06
+          - operand[1].number: 0x07
+          - operand[1].number: 0x08
```

Comment:
```suggestion
```

common array index deltas, remove to avoid FPs?

note this is based on intuition, so feel free to push back/discuss.